modules/introduction/partials/new-features-72.adoc

```diff
@@ -76,3 +76,13 @@ You set how long  Couchbase Server should keep audit logs by using the new `prun
 The default value of 0 means that Couchbase Server does not prune audit logs. 
 See xref:rest-api:rest-auditing.adoc[Configure Auditing].
 
+[#power-bi-connector-1-0-release]
+* Power BI Connector version{nbsp}1.0 released.
```

Comment:
Maybe a link to the xref:power-bi-connector:ROOT:index.adoc[Power BI Connector documentation] ?